src/QsCompiler/QirGeneration/QIR/DataStructures.cs

```diff
@@ -140,7 +140,7 @@ internal SimpleValue(Value value, ResolvedType type)
     internal class PointerValue : IValue
     {
         private readonly IValue.Cached<IValue> cachedValue;
-        private readonly Value pointer;
+        internal readonly Value Pointer;
```

Comment:
This causes a warning that this field should be private. What is the proper way to access this field?